nav/nav_stack/config/caffeine_costmap_common_params.yaml

```diff
@@ -0,0 +1,9 @@
+obstacle_range: 5.5
+raytrace_range: 6.0
+
+footprint: [[-0.734, -0.582], [-0.734, 0.582], [0.734, 0.582], [0.734, -0.582]]
+inflation_radius: 1.0
```

Comment:
Were these values tested such that Caffeine navigates around obstacles without bumping into them? Have you checked that the footprint encloses Caffeine when observing the costmap from the top-down?